macao.py

```diff
@@ -26,7 +26,15 @@ def __init__(self, num_of_players):
             player.draw_initial_hand(self.deck)
 
         # lay out the first card on the table
-        self.table.append(self.deck.pool.pop(random.randint(0, len(self.deck.pool) - 1)))
+        # self.table.append(self.deck.pool.pop(random.randint(0, len(self.deck.pool) - 1)))
+        correct_table = False
+        while not correct_table:
+            drawn_card = self.deck.pool.pop(random.randint(0, len(self.deck.pool) - 1))
+            if drawn_card in ("Ace", "King", "Queen", "Jack", "Four", "Three", "Two"):
+                self.deck.append(card_to_be_drawn)
```

Comment:
please change to "self.deck.pool.append(drawn_card)"

or i.e. "card_to_be_laid_out" if u wish